chacra/async/recurring.py

```diff
@@ -53,6 +54,33 @@ def poll_repos():
     logger.info('completed repo polling')
 
 
+@shared_task(base=base.SQLATask)
+def purge_repos(_now=None):
+    """
+    Purge built repositories, including the associated model objects.
+    """
+    if getattr(pecan.conf, 'purge_repos', False) is False:
+        logger.info('purge_repos option is unset or explicitly disabled, will skip purge')
+        return
+
+    logger.info('polling repos for purging....')
+    now = _now or datetime.datetime.utcnow()
+
+    # XXX Magical datetime, not user configurable
+    lifespan = now - datetime.timedelta(weeks=2)
```

Comment:
that is what I was told initially by @athanatos 

We can modify this easily if we need to I think